calico-cloud/image-assurance/understanding-scan-results.mdx

```diff
@@ -59,6 +59,18 @@ To change the CVSS threshold values, note the following:
 - Changes to threshold values take effect immediately and alter the scan results for images already in the system
 - If you are using admission controller policies, changing a value may allow pods in your Kubernetes cluster that were previously being blocked, to now be deployed or vice versa.
 
+## Exploit Data
+
+The scanning process also attaches EPSS and known exploit data onto each image and vulnerability viewable through the UI.
+
+An EPSS score of 0.1 - or 10% - means a vulnerability has a 10% probability that the vulnerability will be exploited in the wild within the next 30 days. 
+Users should use this information alongside CVSS scores to prioritize remediating vulnerabilities. 
+
+Note that an EPSS score of > 10% can be considered a high number.
+Information about [The EPSS Model](https://www.first.org/epss/model) can be found on the EPSS website created by [FIRST](https://www.first.org/).
+
+Known exploits are based off of the [CISA KEV Catalog](https://www.cisa.gov/known-exploited-vulnerabilities-catalog).
```

Comment:
Could use another sentence here to clarify what an known exploit means for a CVE and how the user should use that information (even if it means we basically say "use known exploits to prioritize remediation efforts" just like EPSS score).